projects/components/src/table/cells/csv-generators/table-cell-boolean-csv-generator.ts

```diff
@@ -0,0 +1,13 @@
+import { TableCellCsvGeneratorBase } from '../table-cell-csv-generator-base';
+import { Injectable } from '@angular/core';
+import { isUndefined } from 'lodash-es';
+
+@Injectable({ providedIn: 'root' })
+export class TableCellBooleanCsvGenerator implements TableCellCsvGeneratorBase<boolean> {
+  public readonly type = 'CSV_GENERATOR';
+  public readonly cellType: string = 'boolean';
+
+  public generateSafeCsv(cellData?: boolean): string | undefined {
```

Comment:
how confident are we that every value passed to this will be boolean|undefined? do we enforce it? Stray nulls etc. often cause issues with these type of contracts.

Reply:
we should make sure the generate method accounts for null/undefined. updated all existing generators to account fo rthis.